app/scripts/lib/get-first-preferred-lang-code.js

```diff
@@ -37,9 +37,15 @@ export default async function getFirstPreferredLangCode() {
     .find(
       (code) =>
         existingLocaleCodes[code] !== undefined ||
-        existingLocaleCodes[code.split('-')[0]] !== undefined,
+        existingLocaleCodes[code.split('-')[0]] !== undefined ||
+        code === 'zh',
```

Comment:
Could we move this to a separate PR? This seems like a distinct change.

Reply:
removed in latest commit and opened a new PR [here](https://github.com/MetaMask/metamask-extension/pull/16329)

Reply:
Thanks!